DigitalLearningSolutions.Data/DataServices/RegionDataService.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace DigitalLearningSolutions.Data.DataServices
+{
+    using System.Collections.Generic;
+    using System.Data;
+    using Dapper;
+
+    public interface IRegionDataService
+    {
+        public IEnumerable<(int, string)> GetRegionsAlphabetical();

```

Comment:
I wonder if it's worth naming Item1 & Item2 so if we ever use it elsewhere the tuple is nicer to use (not feeling strongly either way)
`public IEnumerable<(int regionId, string regionName)> GetRegionsAlphabetical();`